draft-ietf-cats-framework.md

```diff
@@ -172,10 +174,14 @@ Service site:
  : A CATS-serviced site is a service site that is connected to a CATS-Forwarder.
 
 Service contact instance:
-  : A client-facing service function instance that is responsible for receiving requests in the context of a given service. A service request is processed according to the service logic (e.g., handle locally or solicit backend resources). Steering beyond the service contact instance is hidden to both clients and CATS components.
-  : a service contact instance is reachable via at least one Egress CATS Forwarder.
+  : A client-facing service function instance that is responsible for receiving requests in the context of a given service. A service contact instance can represent one or more service instance. Steering beyond the service contact instance is hidden to both clients and CATS components. A service request is processed according to the service logic (e.g., handle locally or solicit backend resources).
```

Comment:
```suggestion
  : A client-facing service function instance that is responsible for receiving requests in the context of a given service.
  : A service contact instance can handle one or more service instances.
  : Steering beyond a service contact instance is hidden to both clients and CATS components.
  : A service request is processed according to the service logic (e.g., handle locally or solicit backend resources).
```

"A service contact instance can represent one or more service instance." seems to be redundant with "A service contact instance may dispatch service requests to one or more service instances". 

Also, I don't parse what is meant by "represent"? Do you mean "A service contact instance can handle one or more service instances."?